stage_descriptions/auth-07-nm2.md

```diff
@@ -0,0 +1,79 @@
+In this stage, you'll add support for enabling a newly created user using the `ACL SETUSER` command with the `on` flag.
```

Comment:
Can we place this before the passwords stage? In the GETUSER response the order is flags first and passwords next. Unless we NEED passwords to enable a user?